model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -19,19 +20,26 @@
 
 package org.ossreviewtoolkit.model.config
 
+import com.fasterxml.jackson.annotation.JsonAlias
+
 data class AnalyzerConfiguration(
     /**
      * If set to true, ignore the versions of used command line tools. Note that this might lead to erroneous
      * results if the tools have changed in usage or behavior. If set to false, check the versions to match the
      * expected versions and fail the analysis on a mismatch.
      */
-    val ignoreToolVersions: Boolean,
+    val ignoreToolVersions: Boolean = false,
```

Comment:
If you set the default value you should add "Defaults to `false`" to the docs of the property.
This and the change below should go into a separate commit.